cluster/addons/rbac/cluster-loadbalancing/glbc/roles.yaml

```diff
@@ -55,5 +57,9 @@ rules:
   resources: ["backendconfigs"]
   verbs: ["get", "list", "watch", "update", "create", "patch"]
 - apiGroups: ["networking.gke.io"]
-  resources: ["servicenetworkendpointgroups"]
+  resources: ["servicenetworkendpointgroups","gcpingressparamses"]
```

Comment:
`gcpingressparamses`? is it too late to fix this to be `gcpingressparams`?

Reply:
The GCPIngressParams CRD has already been merged: https://github.com/kubernetes/ingress-gce/pull/1318

Reply:
But no tag or release has been cut with that commit.

Reply:
well... CRDs give you that control, and the [client generator does as well](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/code-generator/cmd/client-gen/args/args.go#L92), but I'm not sure how to thread the custom `--plural-exceptions=GCPIngressParams:GCPIngressParams` through the `generate_groups.sh` script

@sttts, is there a way to indicate a plural override via a tag in types.go?

Reply:
Thanks for the reference @liggitt! I was able to pipe that in `generate_groups.sh`.